python/Makefile

```diff
@@ -1,15 +1,17 @@
-.PHONY: licenses install-dev test build push-test push
+.PHONY: install-dev check-dev licenses test build push-test push
 
-licenses:
+install-dev:
```

Comment:
IIRC this `Makefile` (and the `README.md` as well) gets generated from the `./templates/python` folder - so you'll want to also add the changes there (and then call `make python` to confirm it works as expected!).

Reply:
🤦🏻‍♂️ Done!